recipes/relion/meta.yaml

```diff
@@ -0,0 +1,62 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "RELION" %}
+{% set version = "3.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/3dem/relion/archive/refs/tags/{{ version }}.tar.gz
+  sha256: f10934058e0670807b7004899d7b5103c21ad77841219352c66a33f98586d42e
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - cmake
+    - make
+  host:
+    - {{ mpi }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - libtiff
+    - fltk
+    - fftw
+  run:
+    - {{ mpi }}
+    - python
+
+test:
+  commands:
+    - test -f $PREFIX/bin/relion  # [unix]
```

Comment:
Does `relion` have a `--help` command-line argument or similar? It might be more informative to actually run something that just test its existence, in case compilation produced something, but something wrong :D

Reply:
it does indeed! Below is the output for the latest version - will add a test 🙂 

```txt
(relion-devel) hex.lmb.internal ➜  processing  relion --help
 [--refresh 2]  : refresh rate in seconds
 [--idle 3600]  : quit GUI after this many second
 [--readonly]   : limited version of GUI that does not touch any files
 [--tomo]       : show tomography-specific GUI
 [--ccpem]      : use the ccpem pipeliner
 [--do_projdir] : Don't confirm the creation of a new project directory, just make it if it doesn't exist
 [--version]    : show the version of this program
```